### PR TITLE
Added Navbar and Color toggle

### PR DESCRIPTION
**Pull request by nanarboursal**
Screen Shot 2020-07-21 at 7 13 40 PM

### CODE REVIEW

**Comment by nanarboursal** [comment]
I'll edit the navbar as more pages are merged in!

**Comment by seemavora** [comment]
We should at least add the dropdown to include the different disabilities.

File: src/components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+#logo-image {
+  width: 30px;
+}
+
+.drp-item.active, .drp-item:active {
+  background-color:#d3d3d3 !important;
+  outline: none !important;
+}
```
Comment (seemavora) [inline src/components/navbar/navbar.css]: Need a new line!

**Review by Annabel1054 — COMMENTED** [review]
This is a great start :)) I would suggest adding a space between the foot emoji and the website name! Also, I agree with Seema. Maybe when the user clicks the website name, have a drop down that will appear which includes activities, rewards, etc! (But only have that toggle if the user is signed in)

**Comment by ThaiQ** [comment]
Instead of dropdown why not like a switch (dropdown is good too). Is yellow a bit too bright compared to blue?

**Comment by surabhigupta2029** [comment]
This looks good nanar! I agree with Thai, you should maybe try using a different text color because the yellow might be a little irritating on a light blue background. And I was thinking instead of a dropdown, do you guys want to have two big buttons or like components on the homepage, one for ADHD and one for Autisim?

**Review by seemavora — APPROVED** [review]

**Review by ThaiQ — APPROVED** [review]

**Review by Annabel1054 — APPROVED** [review]

**Pull request merged**